elide-swagger/src/main/java/com/yahoo/elide/swagger/OpenApiBuilder.java

```diff
@@ -248,28 +254,46 @@ public PathItem getRelationshipPath() {
             RelationshipType relationshipType = dictionary.getRelationshipType(parentClass, name);
 
             if (relationshipType.isToMany()) {
-                path.get(new Operation().tags(getTags()).description("Returns the relationship identifiers for " + name)
-                        .responses(new ApiResponses().addApiResponse("200", okPluralResponse)));
+                if (canCreate(type)) {
```

Comment:
Similar to GraphQL, when you are POSTING to a relationship, we are not creating the type but just updating the relationship (which is Update permission).  All of the relationship edits are effectively Updates.

Reply:
Have made the changes.